web/src/Root.tsx

```diff
@@ -306,6 +306,9 @@ const Root = () => {
   };
 
   const ping = async (tries = 0) => {
+    if (!Utilities.isLoggedIn()) {
```

Comment:
temp fix. block the ping request if there is no token

`isLoggedIn` is not actually checking a login state it's trying to fetch a token. if the token is not there we assume this request won't work. since ping is executed in an interval we don't want to trigger a logout from the interval running while the user is logged out.